api/src/db/DynamoBusinessesDataClient.ts

```diff
@@ -1,27 +1,19 @@
-/* eslint-disable @typescript-eslint/no-unused-vars */
-/* eslint-disable @typescript-eslint/no-explicit-any */
-
-import { ExecuteStatementCommand, QueryCommand, QueryCommandInput } from "@aws-sdk/client-dynamodb";
+import { AttributeValue, QueryCommand, QueryCommandInput } from "@aws-sdk/client-dynamodb";
 import { DeleteCommand, DynamoDBDocumentClient, GetCommand, PutCommand } from "@aws-sdk/lib-dynamodb";
 import { unmarshall } from "@aws-sdk/util-dynamodb";
-import { migrateData } from "@db/config/dynamoDbConfig";
 import { BusinessesDataClient } from "@domain/types";
+import { LogWriterType } from "@libs/logWriter";
 import { Business } from "@shared/userData";
 
 export const DynamoBusinessesDataClient = (
```

Comment:
[dust] This should be singular to be inline with the way we've named the `DynamoUserDataClient`
```suggestion
export const DynamoBusinessDataClient = (
```